data/ashp/ashp_defaults.json

```diff
@@ -0,0 +1,30 @@
+{   
+    "SpaceHeating":
+    {
+        "installed_cost_per_ton": 2250,
+        "om_cost_per_ton": 40,
+        "macrs_option_years": 0,
+        "macrs_bonus_fraction": 0.0,
+        "can_supply_steam_turbine": false,
+        "can_serve_process_heat": false,
+        "can_serve_dhw": false, 
+        "can_serve_space_heating": true,
+        "can_serve_cooling": true,
+        "force_into_system": false,
+        "back_up_temp_threshold": 10.0
```

Comment:
@atpham88 can you append "_degF" to this field name? That's what we do for other temperature inputs.

Reply:
done here https://github.com/NREL/REopt.jl/pull/372/commits/ce7b6826c8009a22999d30e9e203d91a176631c4